### DISCUSSION TITLE
Setting modifiers inside Docker

### DISCUSSION THREAD

**Discussion by travpeg** [lloesche/valheim-server-docker#664] [General]
After playing on my friend's G-Server for a bit (and getting hooked on this game), I decided to try to run a server off my Synology NAS for my kiddos.

I'm relatively new to running a Valheim server, and read over a bit of the documentation for this container to get started.  Fired up after making a couple firewall changes, and all was right with the world....

That is until I got jumped out in the woods and forgotten that I had not eaten anything....  The Gods were Swift (not Merciful).  

- But I also noticed when I re-spawned, that I had nothing equipped.  This is different than when I was playing on my friend's server, where I would keep the items i had equipped when I died.
- A few hours later, we were mining for copper, when i noticed an icon in my inventory saying I could not use a portal with this item.  Again, not the setting on my friend's G-Server I was accustomed to.

So, back to re-reading the manuals, and settings for adjusting the modifers in-game....

https://github.com/lloesche/valheim-server-docker
https://github.com/lloesche/valheim-server-docker/files/12820591/Valheim.Dedicated.Server.Manual.pdf

I put this in place, and it did fix the issue of the portals and ore....
SERVER_ARGS   -modifier "portals=casual"

However, when I tried to put this in place, it does not appear that the death penalty has been modified.
-modifier "portals=casual,deathpenalty=casual,raids=less"

I cannot have multiple "SERVER_ARGS" defined, as this obviously throws an error.  Am I not following correct syntax for putting this on the same line?

![image](https://github.com/lloesche/valheim-server-docker/assets/163423249/49711f7b-de0f-48f5-a38a-881749fc800f)

**Comment by travpeg** [comment]
So, I noticed in my docker attributes while it was running that there was a tag for:
-modifier portals casual

I should have taken a screenshot of it but I didn't...  It seems like no matter what I did once I stopped the server and changes the "SERVER_ARGS" values, this was always constant, and could not be changed, even when I removed this attribute from my current environment variables.  Essentially, this appeared to be "sticky"

Good news though....

I did add the below change for deathpenalty, and this appears to have finally worked on the server.

SERVER_ARGS -modifier deathpenalty casual

I'm still unsure of why I was not able to add multiple SERVER_ARGS values at the same time, but this appears to be running the way I wanted now....

- I can take ore through portals
- I keep my equipped items upon death

![image](https://github.com/lloesche/valheim-server-docker/assets/163423249/99ba36dc-968b-4cff-9b35-87b32d3c6cd2)

**Reply by LiaD1987** [reply]
Did you ever figure this out? Struggling with the same problem.

**Comment by diongerritzen** [comment]
To add multiple SERVER_ARGS, simply put the flags after each other. So for example like this:
`-modifier combat easy -modifier deathpenalty veryeasy -modifier resources muchmore`
